packages/backend-app-api/src/config/config.ts

```diff
@@ -88,6 +88,7 @@ export async function loadBackendConfig(options: {
   const { appConfigs } = await loadConfig({
     configRoot: paths.targetRoot,
     configTargets: configTargets,
+    disableConfigWatching: !!args?.['disable-config-watching'],
```

Comment:
I tend to prefer `Boolean(...)`, feels a bit cleaner than `!!...`

Reply:
Done!